Libraries/src/Amazon.Lambda.Serialization.Json/JsonNumberToDateTimeDataConverter.cs

```diff
@@ -12,6 +12,7 @@ namespace Amazon.Lambda.Serialization.Json
     /// </summary>
     internal class JsonNumberToDateTimeDataConverter : JsonConverter
     {
+        private const long YEAR_5000_IN_SECONDS = 157753180800;

```

Comment:
When checking the math at first I assumed this was seconds-since-epoch, but appears to be since year 1. https://dotnetfiddle.net/0VesSV

Not a big deal functionally though, same principle applies.

Reply:
Added a comment to make it clearer.